truss/remote/baseten/remote.py

```diff
@@ -138,9 +138,10 @@ def _prepare_push(
         if model_name.isspace():
             raise ValueError("Model name cannot be empty")
 
-        gathered_truss = TrussHandle(truss_handle.gather())
+        if truss_handle.is_scattered():
```

Comment:
Small cleanup, no need to recreate the entire TrussHandle (which reloads the YAML) if we're not scattered